notebooks/concepts/cubeviz_gif_ndarray_wfc3ir.ipynb

```diff
@@ -0,0 +1,344 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f07603dd",
+   "metadata": {},
+   "source": [
+    "# Cubeviz with unusual data\n",
+    "\n",
+    "This concept notebook shows Cubeviz loading:\n",
+    "\n",
+    "  * GIF animation as a playable cube,\n",
+    "  * plain Numpy array, and\n",
+    "  * HST WFC3/IR ramp FITS file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0adf0be6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jdaviz import Cubeviz"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "014f2ab7",
+   "metadata": {},
+   "source": [
+    "## From GIF\n",
+    "\n",
+    "The GIF is from https://media4.giphy.com/media/J5pnZ53pj4cmu30Rx5/giphy.gif. This needs `imageio` and Pillow to be installed as well."
```

Comment:
Cannot really use `astropy.utils.data.download_file` here because that does not return the extension (all you get is `content`) and I rely on file extension to check for it for now. Not bothered with sophisticated checks since this will not have wide scientific application.